source/partials/_footer.html.erb

```diff
@@ -0,0 +1,7 @@
+<footer>
+    <ul class="footer">
+        <li class="footer__link"><%= link_to '860-406-1021', 'tel:#{8604061021}' %></li>
+        <li class="footer__link"><%= mail_to 'jstolz123@gmail.com', 'jstolz123@gmail.com' %></li>
```

Comment:
You don't have to do this, but for sharing my personal email, I use an email address obfuscator to prevent robots from scraping my email. Something like this: https://docs.joeworkman.net/general/tips/email-obfuscation (HTML Encoding)